### ISSUE TITLE
Mosek 10 fails on exponential cone program

### ISSUE THREAD

**Issue by phschiele** [cvxpy/cvxpy#1944]
**Describe the bug**
The below exponential cone program gives a wrong result when running it with Mosek 10, but works fine with Mosek 9. 

**To Reproduce**
```py
from cvxpy.constraints import ExpCone
import cvxpy as cp

x = cp.Variable()
y = cp.Variable()

obj = cp.Maximize(x + y)
constraints = [
    y >= 0,
    ExpCone(x, y, 1)
]

problem = cp.Problem(obj, constraints)
problem.solve(cp.MOSEK)
print("status:", problem.status)
print("optimal value", problem.value)
print("optimal var", x.value)
print("optimal var", y.value)
```
gives
```py
status: optimal
optimal value -0.0
optimal var -0.0
optimal var -0.0
```

**Expected behavior**
The correct result for this problem is $x=0, y=1$.

![image](https://user-images.githubusercontent.com/44360364/201542656-e79d1f0d-6a27-4586-ab7c-a281fd231984.png)

The correct result is obtained when removing the (redundant) constraint `y >= 0` or changing it to `y >= 0.00001`

**Version**
 - OS: Tested on Ubuntu 20.04 and MacOs Ventura
 - CVXPY Version: 1.3

**Additional context**
I have checked the release notes for Mosek 10, and there was no mention of breaking changes to the exponential cone specifically, so I think the problem is not with the interface.

@aszekMosek Are you aware of any changes that would explain this behavior?

CC: @ericlux

**Comment by aszekMosek** [comment]
A bug in presolve. There will be a fix.

**Comment by aszekMosek** [comment]
We released version 10.0.28 which should fix it.

**Comment by phschiele** [comment]
@aszekMosek awesome, many thanks! I can confirm the fix worked.
